recipes/dj-static/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set package = "dj-static" %}
```

Comment:
Right - so the md5, for sure, and the version.  Thanks.  Still don't understand the package name.